src/vs/workbench/api/browser/mainThreadEditors.ts

```diff
@@ -116,6 +127,75 @@ export class MainThreadTextEditors implements MainThreadTextEditorsShape {
 		return result;
 	}
 
+	private _getTextEditorDiffInformation(textEditor: MainThreadTextEditor): IObservable<ITextEditorDiffInformation | undefined> {
+		const codeEditor = textEditor.getCodeEditor();
+		if (!codeEditor) {
+			return constObservable(undefined);
+		}
+
+		// Check if the TextModel belongs to a diff editor
+		const diffEditors = this._codeEditorService.listDiffEditors();
+		const [diffEditor] = diffEditors.filter(d =>
```

Comment:
Use https://github.com/microsoft/vscode/blob/44bb717c092b693ad3e4eb69c94978fadf0c2624/src/vs/editor/common/config/editorOptions.ts#L55 instead

Reply:
I looked into that, and while I can use the editor options to determine whether the editor is in a diff editor I still need to get a hold of the model of the diffEditor in order to access the text model of the `original` side. Let me know if I am missing something.